batcher/aligned-batcher/src/main.rs

```diff
@@ -63,7 +52,7 @@ async fn main() -> Result<(), BatcherError> {
         }
     });
 
-    metrics::batcher_started();
+    batcher.metrics.batcher_started.inc();
```

Comment:
This should call the function that has a short timer before increasing the value